src/Core/Settings.h

```diff
@@ -86,6 +86,7 @@ class IColumn;
     M(UInt64, azure_max_single_part_upload_size, 100*1024*1024, "The maximum size of object to upload using singlepart upload to Azure blob storage.", 0) \
     M(UInt64, s3_max_single_read_retries, 4, "The maximum number of retries during single S3 read.", 0) \
     M(UInt64, azure_max_single_read_retries, 4, "The maximum number of retries during single Azure blob storage read.", 0) \
+    M(UInt64, azure_max_unexpected_write_error_retries, 4, "The maximum number of retries in case of unexpected errors during Azure blob storage write", 0) \
```

Comment:
may be just "azure_max_single_write_retries" for consistency with other such settings?

Reply:
I kept is same as '`s3_max_unexpected_write_error_retries`' which I think is used in similar places. We use this for write exception in multipart upload, so not sure if its would be good to call it `single_write_retries`

Reply:
Ah, did not see it before, now I see, indeed it is better then.